app/helpers/state_file/nj_tenant_eligibility_helper.rb

```diff
@@ -1,18 +1,18 @@
 module StateFile
   module NjTenantEligibilityHelper
     INELIGIBLE = :ineligible
-    UNSUPPORTED = :unsupported
+    WORKSHEET = :worksheet
     ADVANCE = :advance
 
     class << self
       def determine_eligibility(intake)
         is_ineligible = intake.tenant_home_subject_to_property_taxes_no? || (intake.tenant_building_multi_unit_yes? && intake.tenant_access_kitchen_bath_no?)
-        is_unsupported = intake.tenant_more_than_one_main_home_in_nj_yes? || intake.tenant_shared_rent_not_spouse_yes?
+        requires_applicant_worksheet = intake.tenant_more_than_one_main_home_in_nj_yes? || intake.tenant_shared_rent_not_spouse_yes?
```

Comment:
[dust] might be nice to name this variable the same `requires_worksheet` as the other matching file.

not a biggie but seeing them side by side not quite lining up tickles my brain lol